Utilities/Source/RobotData/RobotData.cpp

```diff
@@ -27,7 +27,7 @@ double RobotData::length() const
 
     for (std::size_t i = 0; i < NUMBER_OF_COORDINATES; ++i)
     {
-        result += coordinates.at(i) * coordinates.at(i) / (TRANSFER_NUMBER * TRANSFER_NUMBER);
+        result += 1.0 * coordinates.at(i) * coordinates.at(i) / (TRANSFER_NUMBER * TRANSFER_NUMBER);
```

Comment:
Why multiply by one?

Reply:
Because of right-hand side expression is `int`. Just lost accuracy. And some tests may cause an signed overflow. It leads to UB.